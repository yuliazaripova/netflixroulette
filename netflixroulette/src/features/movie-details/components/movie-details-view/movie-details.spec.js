import * as React from 'react';

import MovieDetailsView from '.';

describe("MovieDetailsView component", () => {
  it("should render MovieDetailsView component", () => {
    const testProps = {
      data: {
        title: "test title",
        vote_average: 1,
        release_date: "1977-05-25",
        poster_path: "test poster",
        overview: "test overview",
        runtime: 1,
        genres: ["test genre1", "test genre2"],
      }
    }
    const component = shallow(<MovieDetailsView data={testProps}/>);
    expect(component).toMatchSnapshot();
  })
})
