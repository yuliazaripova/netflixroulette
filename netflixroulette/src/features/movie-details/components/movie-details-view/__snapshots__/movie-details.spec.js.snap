// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetailsView component should render MovieDetailsView component 1`] = `
<div
  className="movie-details"
>
  <Poster
    classes="movie-details_poster"
  />
  <div
    className="movie-details_details"
  >
    <h2
      className="movie-details_title"
    />
    <span
      className="movie-details_vote"
    />
    <p>
      <span
        className="movie-details_date"
      >
        NaN
      </span>
      <span
        className="movie-details_runtime"
      >
         min
      </span>
    </p>
    <p
      className="movie-details_overview"
    />
  </div>
</div>
`;
